.docker/nginx-datahub-mock.conf

```diff
@@ -0,0 +1,40 @@
+server {
+  listen 80;
+  listen 443;
+  listen 3002;
+
+  server_name mock-server;
+  root /www/test-data/;
+
+  location / {
+    proxy_pass http://127.0.0.1:3002;
+  }
+
+  location /enhanced-preprints/docmaps/v1/index {
+    alias /www/test-data/docmaps/dummy-index.json;
+  }
+
+  location ~ /enhanced-preprints/docmaps/v1/by-publisher/elife/get-by-doi(.*)000001 {
+    alias /www/test-data/docmaps/dummy-1.json;
+  }
```

Comment:
Possible improvement would be a single location to catch all 3 of these so that adding another docmap would just mean modifying the file and not this nginx.

```
location ~ ^/enhanced-preprints/docmaps/v1/by-publisher/elife/get-by-doi/.*([1-9][0-9]*)$ {
    alias /www/test-data/dummy-$1.json;
  }
```